code/modules/mob/living/carbon/examine.dm

```diff
@@ -306,6 +306,9 @@
 	if(reagents.has_reagent("teslium"))
 		msg += "[p_they(TRUE)] [p_are()] emitting a gentle blue glow!\n"
 
+	if(HAS_TRAIT(src, TRAIT_PLAGUE_ZOMBIE)) //to tell plague zombies easier through clothing
+		msg += "[p_they(TRUE)] smell like rot and death!\n"
```

Comment:
This should make sure it shows up properly for he/she and they
"they smell like rot and death!"
"he smells like rot and death!"
```suggestion
		msg += "[p_they(TRUE)] smell[p_s()] like rot and death!\n"
```